cmd/gitops/upgrade/cmd.go

```diff
@@ -70,6 +68,15 @@ func upgradeCmdRunE() func(*cobra.Command, []string) error {
 		fluxClient := flux.New(osys.New(), &runner.CLIRunner{})
 		factory := services.NewFactory(fluxClient, log)
 
+		wegoConfig, err := kubeClient.GetWegoConfig(ctx, namespace)
+		if err != nil {
+			return fmt.Errorf("failed getting wego config")
+		}
+
+		if upgradeCmdFlags.ConfigRepo == "" {
```

Comment:
Probably we can remove this if statement as there is no config-repo flag for this command.